content/en/serverless/azure_container_apps/_index.md

```diff
@@ -14,8 +14,14 @@ further_reading:
 ## Overview
 Azure Container Apps is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Container Apps through the [Azure integration][1]. Datadog also provides a solution for instrumenting your Container Apps applications with a purpose-built Agent to enable tracing, custom metrics, and direct log collection.
 
+{{< img src="serverless/azure_container_apps/aca_top.png" alt="Datadog UI, Serverless Monitoring page with Azure Container Apps selected." style="width:100%;" >}}
+
 ## Setup
 
+### Azure integration
+
+Install the [Datadog-Azure integration][1] to collect metrics and logs.
```

Comment:
Based on the link text, it doesn't seem to be intentional for this to open the log collection section. Could you confirm if that's expected?